client/src/app/pages/main-feeds/header/Header.tsx

```diff
@@ -19,19 +20,21 @@ const Header = () => {
         </div>
         <div className="header__right">
             <div className="header__user">
-                <img className='logo' src="http://www.clipartbest.com/cliparts/nTE/BGx/nTEBGxgzc.jpg" alt="avatar" />
-                <h4>Tran Ngoc Tien</h4>
+                <img className='avatar' src={avatarURL} alt="avatar" />
```

Comment:
don't use className with no meaning name like this, please use something like img-avatar